system_tests/forwarder_test.go

```diff
@@ -73,31 +81,35 @@ func initMiniRedisForTest(t *testing.T, ctx context.Context, nodeNames []string)
 	testhelpers.RequireImpl(t, err)
 	defer redisClient.Close()
 
-	for _, name := range nodeNames {
-		priorities = priorities + name + ","
-	}
-	priorities = priorities[:len(priorities)-1] // remove last ","
+	priorities := strings.Join(nodeNames, ",")
+
 	testhelpers.RequireImpl(t, redisClient.Set(ctx, redisutil.PRIORITIES_KEY, priorities, time.Duration(0)).Err())
 	return redisServer, redisUrl
 }
 
-func createFallbackSequencer(
-	t *testing.T, ctx context.Context, ipcPath string, redisUrl string,
+type fallbackSequencerOpts struct {
```

Comment:
Advantage is that it's more readable. Rather than having

funcName(arg1, arg2, ....true, 0,., arg37)  (where it's hard to understand what does each argument represent)

You have 

funcName( &funcNameOpts{
    ipcPath: arg1,
    redisURL: arg2,
    isThisGood: true,    
 })


Also for more detailed explanation see [0], [1].

[0] https://google.github.io/styleguide/go/best-practices#function-argument-lists
[1] https://google.github.io/styleguide/go/best-practices#option-structure